src/app/documentation/documentation.component.html

```diff
@@ -12,32 +12,32 @@ <h1 class="page-header">Documentation</h1>
                   <h2>PDF Documentation</h2>
                   <p>PDF documentation can be downloaded from the following links.</p>
                   <p>
-                    <a class="btn btn-primary btn-lg btn-bg-dark" role="button" href="assets/nrel_custom/pdfs/pdfs_v9.5.0/pdfs.zip" download>Download All PDFs<img src="assets/images/icon_zip.png" class="figure-img ml-1 mb-1"></a></p>
+                    <a class="btn btn-primary btn-lg btn-bg-dark" role="button" href="assets/nrel_custom/pdfs/pdfs_v9.6.0/pdfs.zip" download>Download All PDFs<img src="assets/images/icon_zip.png" class="figure-img ml-1 mb-1"></a></p>
```

Comment:
Updated all the PDF links to a new folder `pdfs_v9.6.0`.  I didn't delete the 9.5 PDFs but I can do that.